web/core/components/issues/issue-detail-widgets/sub-issues/helper.ts

```diff
@@ -41,7 +41,8 @@ export const useSubIssueOperations = (issueServiceType: TIssueServiceType): TSub
   const subIssueOperations: TSubIssueOperations = useMemo(
     () => ({
       copyLink: (path) => {
-        copyUrlToClipboard(`/${path}`).then(() => {
+        const originURL = typeof window !== "undefined" && window.location.origin ? window.location.origin : "";
```

Comment:
Don't do it like this. Just use `copyUrlToClipboard(path)`, It will take up the relative URL directly. The problem was with `/${path}`